flint.ui/src/components/Footer/Footer.vue

```diff
@@ -1,137 +1,151 @@
-              >Resources</a
-            >
-          </a-typography-title>
-          <a-typography-title>
-            <a href="https://moja.global/news/" target="_blank" style="text-decoration: none; color: white">News</a>
-          </a-typography-title>
-        </a-space>
-      </a-col>
-    </a-row>
-    <a-typography-title
-      style="display: flex; align-items: center; justify-content: center; margin-top: 2rem; color: #7a7a7a"
-    >
-      Copyright © 2022 moja global.
-    </a-typography-title>
-  </Footer>
+            <a-col :xm="7" :sm="7" :md="3" :lg="3" :xl="3">
+                <a-space direction="vertical" size="middle" class="df">
+                    <a-typography-title class="white extra-bold">Site Links</a-typography-title>
+                    <a-typography-title>
+                        <a href="https://moja.global/about/" target="_blank" class="links">About</a>
+                    </a-typography-title>
+                    <a-typography-title>
+                        <a href="https://moja.global/what-we-do/" target="_blank" class="links">What We Do</a>
+                    </a-typography-title>
+                    <a-typography-title>
+                        <a href="https://moja.global/flint/" target="_blank" class="links">FLINT</a>
+                    </a-typography-title>
+                    <a-typography-title>
+                        <a href="https://moja.global/get-involved/" target="_blank" class="links">Get Involved</a>
+                    </a-typography-title>
+                    <a-typography-title>
+                        <a href="https://moja.global/resources/" target="_blank" class="links">Resources</a>
+                    </a-typography-title>
+                    <a-typography-title>
+                        <a href="https://moja.global/news/" target="_blank" class="links">News</a>
+                    </a-typography-title>
+                </a-space>
+            </a-col>
+        </a-row>
+        <a-typography-title class="fc copyright">
+            Copyright © 2022 moja global.
+        </a-typography-title>
+    </Footer>
 </template>
 
 <script>
-export default {
-  name: 'LandingPageNavbar'
-}
+    export default {
+        name: 'FooterComponent'
+    }
 </script>
+
+<style scoped>
+    .footer-container {
+        padding-top: 3.5rem;
+        padding-bottom: 1rem;
+        background-color: #2f382a
+    }
+
+    .justify-center {
+        justify-content: center
+    }
+
+    .fc {
+        display: flex;
+        align-items: center;
+        justify-content: center
+    }
+
+    .white {
+        color: white;
+    }
+
+    .extra-bold {
+        font-weight: 800;
+        font-size: 1.2rem
+    }
+
+    .social-icons {
+        background-color: white;
+        width: 3rem;
+        height: 3rem;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    .footer-container .icon-clr {
+        color: #a6adb4
+    }
+
+    a.links {
```

Comment:
Since we keeping text-decoration as none; considering accessibility in mind - I guess it be a good idea to change the color shade on hover effect for links

Reply:
Just a little bit darker shade would work